maplibre/src/style/style.rs

```diff
@@ -33,6 +35,8 @@ impl Default for Style {
             center: Some([46.5197, 6.6323]),
             pitch: Some(0.0),
             zoom: Some(13.0),
+            maxzoom: Some(20),
+            minzoom: Some( 0), // karlos: ok??? Detault values in extra file?
```

Comment:
Any values are fine here. This whole file is just a placeholder  basically until we have something more sophisticated.